pipmonkey.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+ <html>
+ <head>
+    <title>pipmonkey the codemonkey</title>
+  </head>
+  <body>
+    <h1>About</h1>
+    <p>Call me <em>pipmonkey</em>, brah.</p>
+    <a href="www.google.com">Go to the google homepage</a>
```

Comment:
Can you please prefix the url with http://  😃 
The current url is relative to the page url